.github/workflows/release.yml

```
@@ -34,8 +34,14 @@ jobs:
           npm install --no-package-lock
           npm install --no-save svelte@${{ matrix.svelte }}
 
-      - name: ▶️ Run validate script
-        run: npm run validate
+      - name: ▶️ Test
+        run: npm test
+
+      - name: ▶️ Lint and type-check
+        if: ${{ matrix.node == '20' && matrix.svelte == '4' }}
+        run: |
+          npm run lint
```

As per the other discussion, we can't assume all files will be formatted to prettier's and eslint's current expectations.

```suggestion
      - name: ▶️ type-check
        if: ${{ matrix.node == '20' && matrix.svelte == '4' }}
        run: |
```